options/locale/locale_en-US.ini

```diff
@@ -693,6 +695,23 @@ issues.attachment.open_tab = `Click to see "%s" in a new tab`
 issues.attachment.download = `Click to download "%s"`
 issues.subscribe = Subscribe
 issues.unsubscribe = Unsubscribe
+issues.tracker = Time tracker
+issues.start_tracking_short = Start
+issues.start_tracking = Start time tracking
+issues.start_tracking_history = `started working %s`
+issues.tracking_already_started = `You already stared tracking at this <a href="%[1]s">issue</a>!`
```

Comment:
It should be better `You have already started time tracking on this <a href="%s">issue</a>!`